.pre-commit-config.yaml

```diff
@@ -91,3 +91,9 @@ repos:
         exclude: ".*(.fits|.fts|.fit|.txt|tca.*)$"
       - id: mixed-line-ending
         exclude: ".*(.fits|.fts|.fit|.txt|tca.*)$"
+  - repo: https://github.com/devanshshukla99/pre-commit-hook-prohibit-string
+    rev: v1.2
+    hooks:
+      - id: prohibit-string
+        args: [--prohibit-string, "from warnings import warn,warnings.warn"]
+        exclude: "sunpy/version.py|sunpy/util/decorators.py|sunpy/util/exceptions.py"
```

Comment:
What is in the version file?

Reply:
The top of the file is
```python
# NOTE: First try _dev.scm_version if it exists and setuptools_scm is installed
# This file is not included in sunpy wheels/tarballs, so otherwise it will
# fall back on the generated _version module.
try:
    try:
        from ._dev.scm_version import version
    except ImportError:
        from ._version import version
except Exception:
    import warnings

    warnings.warn(
        f'could not determine {__name__.split(".")[0]} package version; this indicates a broken installation'
    )
    del warnings

    version = '0.0.0'
```